hack/push-chart.sh

```diff
@@ -18,8 +18,8 @@ DEST_CHART_DIR=${DEST_CHART_DIR:-bin/}
 
 EXTRA_TAG=${EXTRA_TAG:-$(git branch --show-current)} 
 GIT_TAG=${GIT_TAG:-$(git describe --tags --dirty --always)}
-HELM_CHART_REPO=${HELM_CHART_REPO:-gcr.io/k8s-staging-kueue/charts}
-IMAGE_REPO=${IMAGE_REPO:-gcr.io/k8s-staging-kueue/kueue}
+HELM_CHART_REPO=${HELM_CHART_REPO:-us-central1-docker.pkg.dev/k8s-staging-images/charts}
```

Comment:
should we do something about 

```
HELM_CHART_REPO := $(IMAGE_REGISTRY)/charts
```
In Makefile L40?

Reply:
Done.